server/src/main/java/org/opensearch/gateway/AsyncShardFetchCache.java

```diff
@@ -193,6 +197,8 @@ private boolean validateNodeResponse(BaseNodeEntry nodeEntry, long fetchingRound
     }
 
     private void handleNodeFailure(BaseNodeEntry nodeEntry, FailedNodeException failure, long fetchingRound) {
+        clusterManagerMetrics.incrementCounter(clusterManagerMetrics.asyncFetchTotalCounter, 1.0);
```

Comment:
- i think we can get the sum from success and failures and need not emit additional metric.
- cumulative count for a fetch round can be emitted in `processFailures`

Reply:
makes sense